testsuite/cluster-test/src/deployment.rs

```diff
@@ -129,12 +130,11 @@ impl DeploymentManager {
             image_digest: Some(hash.clone()),
             image_tag: None,
         }];
-        let response = self
-            .aws
-            .ecr()
-            .batch_get_image(get_request)
-            .sync()
-            .map_err(|e| format_err!("Failed to get image {}: {:?}", hash, e))?;
+        // Retry upto 10 times, waiting 10 sec between retries
+        let response = retry(Fixed::from_millis(10_000).take(10), || {
+            self.aws.ecr().batch_get_image(get_request.clone()).sync()
```

Comment:
I am thinking about use case where for example you ran this code and forgot to give node permissions to run the operation.
In this case your execution will freeze for 100 seconds before you get simple feedback that there is no permissions.

Perhaps easiest way to handle it is to print a message if attempt fails (something like 'XX failed: YYY, retrying') - this would allow us to understand if something going on before `10*10` seconds elapses.

Something similar to [this](https://github.com/libra/libra/blob/69c569517a17e98d7a053ad80d03ad890d5f77a1/testsuite/cluster-test/src/cluster.rs#L64)

Reply:
Updated.